plonecli/cli.py

```diff
@@ -28,13 +29,26 @@ def get_templates(ctx, args, incomplete):
     invoke_without_command=True
 )
 @click.option('-l', '--list-templates', 'list_templates', is_flag=True)
+@click.option('-v', '--versions', 'versions', is_flag=True)
```

Comment:
Please a large "V" not a small "v" 

-V is common for version
-v is common for verbose mode